beta-prompts/beta_prompts/seaborn_analytics.py

```
@@ -0,0 +1,1413 @@
+                "alpha": 0.8,
+            },
+        )
+
+    def _create_performance_distributions(self, ax, data: pd.DataFrame):
+        """Create multi-faceted performance distribution analysis"""
+
+        # Create ridge plot style distribution
+        categories = data["technique_category"].unique()
+        colors = self.neon_palette[: len(categories)]
+
+        y_offset = 0
```

Ruff (0.12.2)

Local variable `y_offset` is assigned to but never used

(F841)